custom-per-route-options.html.md.erb

```diff
@@ -0,0 +1,92 @@
+---
+title: Configuring per-route options
+owner: CF for VMs Networking
+---
+
+By default, the communication between Gorouter and backends can only be configured at the platform level. This means that any settings affecting this communication apply universally across the entire platform.
+However, with the introduction of per-route features, developers now have the flexibility to customize this configuration on a more granular level.
+
+The ability to customize properties on a per-route basis provides greater control and adaptability, ensuring that the route can be optimized for its unique requirements.
+
+The following per-route options exist:
+- `loadbalancing`: Configures the load balancing algorithm used by Gorouter for this particular route. Possible options are: `round-robin` and `least-connections`.
```

Comment:
There is only one bullet in the list. Does it need to be a list or did you miss putting the others in?